### ISSUE TITLE
"The directory given does not exist: "

### ISSUE THREAD

**Issue by sharazam** [quarnster/SublimeGDB#101]
SublimeGDB currently fails to parse a valid `workingdir` entry when using `$` in the string:

```
"workingdir": "${folder:${project_path}}",
```

All I get is an empty popup box:

![pic1](https://user-images.githubusercontent.com/12084016/29465635-44b7523c-843a-11e7-9aaa-4628ec421ad7.png)

It doesn't tell me what path this expanded to and is very uninformative. This is copy-pasted from the user examples. If you, however, do this:

```
"workingdir": ".",
```
... then it suddenly works. So my suspect is that parsing the working dir doesn't work as expected.

OS: Ubuntu 17.04
GDB: GNU gdb (Ubuntu 7.12.50.20170314-0ubuntu1.1) 7.12.50.20170314-git
ST: Sublime Text 3, Build 3126
Python version: Python 2.7.13

**Comment by hsandt** [comment]
Yeah, according to the SublimeGDB preference comments, `${project_path:file}` should search for file in all the project hierarchies. But it doesn't seem to work (`${folder:}` does, though) and is not very useful for my project where the project name is also the name of the executables, and the same in all build configs, so it is too ambiguous.

I ended using `${home}/path/to/my/executable/dir` directly. Not very portable across working machines, unless you stick to the same relative path on all of them.

I would just like the standard `${project_path}` (or if it conflicts with the search pattern, `#{project_dir}`) so I can write the path relatively to the project and make it portable.
